scp/tickets.php

```diff
@@ -218,7 +219,7 @@
                             $assigned, $thisstaff->getName());
                         $ticket->logActivity(__('Ticket unassigned'),$msg);
                     } else {
-                        $errors['err'] = __('Problems releasing the ticket. Try again');
+                        $errors['err'] = sprintf('%s %s', __('Problems releasing the ticket.'), __('Try again!'));
```

Comment:
@greezybacon 
Before changing stuff, I'd like to know if you prefer the phrase `Please try again!` (instead of `Try again!`) just here in `scp/tickets.php` or in all files?

Asking since the statistics are as follows:
`Try again!` is used in 12 files (26 occurrences)
`Please try again` is just used in 1 file (scp/tickets.php, 4 occurrences)

PS: Personally I'd prefer the more polite version with "please".